platform/jobs/edxPlatformAccessibilityMaster.groovy

```diff
@@ -134,8 +134,8 @@ secretMap.each { jobConfigs ->
                    }
                }
            }
-           shell('cd edx-platform')
-           shell('RUN_PA11YCRAWLER=1 ./scripts/accessibility-tests.sh')
+           shell('cd edx-platform \n' +
+                 'RUN_PA11YCRAWLER=1 ./scripts/accessibility-tests.sh')
```

Comment:
Nit: Having multiple shell commands in the same line is creeping me out. Can you put them into separate lines?